File: 2022.1/Programação Orientada a Objetos/lista1python/Ex7.py
maxv = -999999999.0
minv = 999999999.0

while(True):
    x = float(input())
    if(x == 0):
        break

    maxv = max(x,maxv)
    minv = min(x,minv)

print("O maior número da sequência é:", maxv)
print("O menor número da sequência é:", minv)

File: 2022.1/Programação Orientada a Objetos/lista1java/Ex7.java
public class Ex7{
    public static void main(String args[]) throws Exception{
        float x;
        float max = -999999999;
        float min = 999999999;

        while(true){
            x = (float)EntradaTeclado.leDouble();
            if(x == 0) break;

            max = Math.max(x, max);
            min = Math.min(x, min);
        }

        System.out.println("O maior número da sequência é: " + max);
        System.out.println("O menor número da sequência é: " + min);
    }
}
